content/compose/environment-variables/best-practices.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Best practices for working with environment variables in Docker Compose
+description: Explainer on the best ways to set, use, and manage environment variables in
+  Compose
+keywords: compose, orchestration, environment, env file, environment variables
+---
+
+#### Handle sensitive information securely
```

Comment:
The heading levels on this page are off, these should be h2s

Reply:
Yes this was done on purpose. The page itself doesn't have a lot of content so having h2s would look out of place. We do a similar thing in the FAQs, so figured a little bit more rule breaking wouldn't be a problem